Specs/DataSources/ModelVisualizerSpec.js

```diff
@@ -372,15 +373,17 @@ describe(
     });
 
     it("Computes bounding sphere with height reference", function () {
+      scene.globe.terrainProvider = createWorldTerrain();
+
       const time = JulianDate.now();
       const testObject = entityCollection.getOrCreateEntity("test");
       const model = new ModelGraphics({
-        heightReference: HeightReference.CLAMP_TO_GROUND,
+        heightReference: HeightReference.RELATIVE_TO_GROUND,
```

Comment:
Since there's potential for different behavior between the two, should there be a test for both `CLAMP_TO_GROUND` and `RELATIVE_TO_GROUND`?

Reply:
Added specs for each.